### ISSUE TITLE
[Initiative] Creative Guild - Joining

### ISSUE THREAD

**Issue by mattbanks** [madglory/governance#7]
I'm interested in joining the Creative Guild. In terms of expanding the design part of the company, I'd like to be involved from a front-end dev perspective.

**Comment by Polidore22** [comment]
I'm also interested in the Creative Guild. I'd like to either make the podcast part of it or probably even its own Project Guild as well.

**Comment by scottglasser** [comment]
Also interested in the Creative Guild. I love the idea of continuing to grow the design side of the business, and would like to help shape that.

**Comment by sethlouey** [comment]
I'm not interested in the Creative Guild :trollface: JKJK. So, how do we get this party started?

**Comment by Polidore22** [comment]
Tattoos.

**Comment by clarkefoley** [comment]
Ahoy! 
I think this guild needs a charter meeting, happy to make that a quick 10 minute thing. I've been thinking about how we want to grow the design business to go hand in hand with becoming the go-to team for all services related to gaming.

I think we should think about what kind of design work we want to become well known for, and publish relevant posts of our work (as close as possible) to what we are striving to become. 

-Do we want to be known for making teaser and pre order websites? (I'm thinking not-exclusively)
-Seth and Josh nailed the Arkham and Pachinko wireframes. Pachinko wireframes were intellignetly planned out to mirror the UI of the Pachinko home / loading screen / main menu.

This got me thinking...

The creative team successfully designed 6 or 8 different UI / UX's for Odd Networks platforms. We've also worked on multiple game clients, websites, leaderboards, and even chat systems (Parley!).

I motion that we should promote that MadGlory design is the industry leader in just these things;

1. Game client interfaces (think LCU / BossKey) 
2. Leaderboards / stats (think PBR stats) 
3. Chat systems (think Parley)
4. Profile pages (think Overdog)
5. Clans / Guilds / Matchmaking interfaces (think Overdog) 

**TL:DR **
We've already designed and launched a ton of game-like service interfaces, we should show that off in our posts and focus on grabbing that work. Think of all the mobile and AAA titles out there looking for UI/UX work for game interfaces.

Woot!

**Comment by genexp** [comment]
One note: I'd like to know how you guys see this guild overlapping with Outreach.  Like, I had originally assumed the goal of this one was to build our creative business and better explore and utilize our creatative assets.  In contrast, Outreach's job is to figure out how to have an ongoing relationship with 100 game companies. (So outreach is really marketing)

I'm not sure then if the podcast is  something this group would want to take on, or the other one, or if its's something they decide on together.  I dunno.
